mk/spksrc.spk.mk

```diff
@@ -117,13 +117,25 @@ endif
 	@echo maintainer_url=\"$(MAINTAINER_URL)\" >> $@
 	@echo distributor=\"$(DISTRIBUTOR)\" >> $@
 	@echo distributor_url=\"$(DISTRIBUTOR_URL)\" >> $@
+
 ifneq ($(strip $(FIRMWARE)),)
 	@echo firmware=\"$(FIRMWARE)\" >> $@
-else
-  ifneq ($(strip $(TC_FIRMWARE)),)
+else ifneq ($(strip $(OS_MIN_VER)),)
+	@echo os_min_ver=\"$(OS_MIN_VER)\" >> $@
+else ifneq ($(strip $(TC_FIRMWARE)),)
 	@echo firmware=\"$(TC_FIRMWARE)\" >> $@
-  else
+else ifneq ($(strip $(TC_OS_MIN_VER)),)
+	@echo os_min_ver=\"$(TC_OS_MIN_VER)\" >> $@
+else
 	@echo firmware=\"3.1-1594\" >> $@
+endif
+ifneq ($(strip $(OS_MIN_VER)),)
```

Comment:
I just looked at the diff on github. This is redundant. Sorry, when I resolved a merge conflict and forced pushed I missed this.